app/Http/Requests/updateCategoryArticleRequest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Http\Requests;
+
+use App\Models\ArticleCategory;
+use Illuminate\Foundation\Http\FormRequest;
+
+class updateCategoryArticleRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
```

Comment:
your problem is due to your rules use this and it will fix the problem. there is no need for extra database query
```php
return [
        'articleCategoryName' => "required|string|max:255|unique:article_categories,articleCategoryName,{$this->slug},slug",
        'hasComments' => 'sometimes',
        'hasSource' => 'sometimes',
        'hasYoutubeLink' => 'sometimes',
        'hasAuthor' => 'sometimes',
        ];
```